### PR TITLE
Telemetry for styling.

### PR DESCRIPTION
**Pull request by oleg-morozov**
JIRA: TNT-1009

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-ui-kit/src/StylingPicker/StylingPicker.tsx [hunk 1]
```diff
@@ -70,6 +77,7 @@ const StylingPickerCore = <T extends StylingPickerItemContent>(
     }, [currentItemRef, customItems, customItems.length, initiallySelectedItemRef]);
 
     const onItemClick = useCallback((ref: ObjRef) => {
+        onItemSelect(ref);
         setCurrentItemRef(ref);
     }, []);
```
Comment (ivan-nejezchleb) [inline libs/sdk-ui-kit/src/StylingPicker/StylingPicker.tsx:82]: missing onItemSelect in deps?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9459/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11319/ : SUCCESS in 16m 11s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by oleg-morozov** [comment]
extended test

**Comment by oleg-morozov** [comment]
extended check sonar

**Comment by oleg-morozov** [comment]
extended check cypress

**Comment by oleg-morozov** [comment]
extended check plugins

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-dashboard-plugin-tests/33/ : SUCCESS in 13m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9468/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check-extended-sonar aborted.

**Comment by gdgate** [comment]
Build in pipeline check-extended-cypress aborted.

**Comment by oleg-morozov** [comment]
extended check sonar

**Comment by oleg-morozov** [comment]
extended check cypress

**Comment by oleg-morozov** [comment]
extended check plugins

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9470/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11327/ : FAILURE in 19m 13s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-dashboard-plugin-tests/34/ : SUCCESS in 12m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9477/ : SUCCESS in 6s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-check-extended-sonar-PR2895

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9475/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1807/ : SUCCESS in 17m 44s

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dashboardFiltering.spec.ts**|2|-|32.4340s|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|20.8660s|
|✅|**drilling.spec.ts**|2|-|23.0520s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|20.9040s|
|✅|**dashboard.spec.ts**|9|-|58.6830s|
|✅|**pivotTableSizingReset.spec.ts**|15|-|60.0570s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|19.1530s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|36.6190s|
|✅|**headlineChartDrilling.spec.ts**|2|-|15.1100s|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|19.1290s|
|✅|**attributeFilterConfig.spec.ts**|3|-|45.5460s|
|✅|**barChartDrilling.spec.ts**|4|-|18.2990s|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|25.8540s|
|✅|**dateFilter.spec.ts**|9|-|32.7220s|
|✅|**attributeFilterButton.spec.ts**|9|-|40.3920s|
|✅|**pivotTableSizing.spec.ts**|1|-|16.5510s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|18.7830s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tests/284/).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9476/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tests/284/ : SUCCESS in 19m 52s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1801/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9488/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11345/ : SUCCESS in 16m 22s

**Review by ivan-nejezchleb — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2895

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2677/ : SUCCESS in 1m 16s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2978/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1813/ : SUCCESS in 15m 06s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11346/ : SUCCESS in 15m 56s

**Pull request merged**